test/controllers/manage/messages_controller_test.rb

```diff
@@ -379,21 +375,21 @@ class Manage::MessagesControllerTest < ActionController::TestCase
       assert_redirected_to manage_messages_path
     end
 
-    should "not allow access to manage_messages#template" do
-      test_template_failure
-    end
+    # should "not allow access to manage_messages#template" do
+    #   test_template_failure
+    # end
 
-    should "not allow access to manage_messages#template_preview" do
-      test_template_preview_failure
-    end
+    # should "not allow access to manage_messages#template_preview" do
+    #   test_template_preview_failure
+    # end
 
-    should "not allow access to manage_messages#template_update" do
-      test_template_update_failure
-    end
+    # should "not allow access to manage_messages#template_update" do
+    #   test_template_update_failure
+    # end
 
-    should "not allow access to manage_messages#template_replace_with_default" do
-      test_template_replace_with_default_failure
-    end
+    # should "not allow access to manage_messages#template_replace_with_default" do
+    #   test_template_replace_with_default_failure
+    # end
```

Comment:
Are these commented out in anticipations of it being fixed by a later issue? If not we can just remove them. If it is the former then you can merge in

Reply:
They'll be resolved with a future issue. They should have a FIXME but I'm going through each test file manually just to be safe.